hazelcast/src/main/java/com/hazelcast/sql/impl/connector/LocalReplicatedMapConnector.java

```diff
@@ -78,6 +78,8 @@ public Table createTable(
 
         long estimatedRowCount = stores.size() * nodeEngine.getPartitionService().getPartitionCount();
 
+        boolean nativeMemoryEnabled = nodeEngine.getConfig().getNativeMemoryConfig().isEnabled();
```

Comment:
Do we support HD for replicated maps? I thought no.

Reply:
no HD support for replicated maps